src/cryptoadvance/specter/wallet.py

```diff
@@ -624,11 +626,12 @@ def balance_on_address(self, address):
         return sum(balancelist)
 
     def utxo_on_label(self, label):
-        utxo = [tx for tx in self.utxo if self.getlabel(tx["address"]) == label]
+        utxo = [tx for tx in self.utxo if self.utxo_labels_list[tx["address"]] == label]
+
         return len(utxo)
```

Comment:
```suggestion
        return len([tx for tx in self.utxo if self.utxo_labels_list[tx["address"]] == label])
```
What about a one-liner here?
If you don't like it then I would suggest renaming `utxo` to `utxos` and removing the blank line.